openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/MetadataToESConfigForm.tsx

```diff
@@ -44,62 +60,99 @@ export default function MetadataToESConfigForm({
   };
 
   return (
-    <Form layout="vertical">
-      <Form.Item label="caCerts">
+    <Form className="metadata-to-es-config-form" layout="vertical">
+      <Form.Item label={startCase('caCerts')}>
+        <Text className="input-field-descriptions">
+          {t('message.field-ca-certs-description')}
+        </Text>
```

Comment:
Use label props of form items instead of explicitly adding the label.

Reply:
I have used label props for input filed labels but not for switches as they were limiting the ability to style the step component similarly to other steps. It was necessary to achieve that.